+neuron/+stack/hoc_push.m

```diff
@@ -1,12 +1,22 @@
-function hoc_push(value)
+function hoc_push(value, varargin)
 % Push a double, string, Vector, Object or NrnRef to the NEURON stack.
 %   hoc_push(value)
+%   hoc_push(value, string_stack)
+    string_stack = [];
+    if nargin >= 2
+        string_stack = varargin{1};
```

Comment:
I know this is an internal function, but the existence of varargin{1} implies that there could be a varargin{2}, which we'd just ignore here